pytools/__init__.py

```diff
@@ -1510,6 +1519,48 @@ def __str__(self):
 
         return "\n".join(lines)
 
+    def github_markdown(self):
+        """Returns a string representation of the table formatted as
+        GitHub-Flavored Markdown.
+        https://docs.github.com/en/github/writing-on-github/organizing-information-with-tables
+
+        .. doctest ::
+
+            >>> tbl = Table(alignments=['l', 'r'])
+            >>> tbl.add_row([1, '|'])
+            >>> tbl.add_row([10, 20])
+            >>> s = tbl.github_markdown().splitlines()
+            >>> assert s[0] == "1  | \\|"
+            >>> assert s[1] == ":--|---:"
+            >>> assert s[2] == "10 | 20"
```

Comment:
You could just print the output and have the doctest make sure it matches. (That's its job.)

Reply:
Fixed.